src/System.CommandLine/Option.cs

```diff
@@ -75,7 +75,7 @@ public ArgumentArity Arity
         /// Global options are applied to the command and recursively to subcommands.
         /// They do not apply to parent commands.
         /// </summary>
```

Comment:
```suggestion
        /// When set to true, this option will be applied to the command and recursively to subcommands.
        /// It will not apply to parent commands.
        /// </summary>
```